features/GEM052_Correct-geometric-subcontexts.feature

```diff
@@ -0,0 +1,20 @@
+@implementer-agreement
+@GEM
+@version1
+@E00050
+Feature: GEM052 - Correct geometric subcontexts
+The rule verifies that there is a minimum of at least one subcontext per context. The context
+identifier should be one of the agreed values:
+
+Scenario: Each geometric context must have a subcontext
+    
+    Given an IfcGeometricRepresentationContext without subtypes
+    
+    Then HasSubContexts = not empty
+
+
+Scenario: Constraints on context type
+    Given An IfcGeometricRepresentationSubContext
+
+    Then ContextIdentifier = 'Annotation' or 'Axis' or 'Box' or 'Footprint' or 'Reference' or 'Body' or 'Clearance' or 'CoG' or 'Profile' or 'SurveyPoints' or 'Lightning'
```

Comment:
```suggestion
    Then ContextIdentifier = 'Annotation' or 'Axis' or 'Box' or 'Footprint' or 'Reference' or 'Body' or 'Clearance' or 'CoG' or 'Profile' or 'Lighting'
```

I don't see `SurveyPoints` in the 4.3 docs.  Was it valid for an earlier schema version?  Per [CT 4.1.7.1.2.3](https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/concepts/Product_Shape/Product_Geometric_Representation/Annotation_Geometry/Set_Of_Survey_Points/content.html) it would fall under `Annotation`.